lametro/models.py

```diff
@@ -236,6 +233,18 @@ def committee_sponsorships(self):
 
         return qs
 
+    @classmethod
```

Comment:
It feels more conventional to define as a method on a Person manager, rather than on the Person class itself. What was your thinking with the `classmethod`?

Reply:
I suppose this seemed more conventional to me.

1. I think about methods on managers returning querysets (I would expect that `LAMetroPerson.objects.ceo()` would return a queryset)
2. seems similar to `upcoming_board_meeting` or `current_meeting` on the LAMetroOrganization classes.

Reply:
Ah, ok! Sure, that makes sense to me.